Cannot find module 'wordwrap' with cli usage

```
npx wikipedia-tldr breckenridge

npx: installed 2 in 2.218s
Cannot find module 'wordwrap'
Require stack:
- /Users/dawsonbotsford/.npm/_npx/98390/lib/node_modules/wikipedia-tldr/cli.js
```

### shell

`uname`: Darwin
`echo $SHELL`: /bin/zsh
`echo $TERM`: xterm-256color
`echo $TERM_PROGRAM`: iTerm.app

### node

`npm -v`: 6.14.11
`node --version`: v14.16.0

--- a/cli.js
+++ b/cli.js
@@ -11,7 +11,6 @@ async function main () {
   }
   const result = await lookup(query)
 
-  console.log(result)
   if (result) {
     console.log(`\x1b[33m%s\x1b[0m`, wrap(`${result.title}: ${result.description}`))
     console.log(wrap(result.extract))

--- a/cli.js
+++ b/cli.js
@@ -10,8 +10,11 @@ async function main () {
     return
   }
   const result = await lookup(query)
+
+  console.log(result)
   if (result) {
-    console.log(wrap(result.text))
+    console.log(`\x1b[33m%s\x1b[0m`, wrap(`${result.title}: ${result.description}`))
+    console.log(wrap(result.extract))
   } else {
     console.log('no results :[')
   }

--- a/package.json
+++ b/package.json
@@ -27,6 +27,7 @@
     ]
   },
   "dependencies": {
-    "node-fetch": "^2.6.1"
+    "node-fetch": "^2.6.1",
+    "wordwrap": "^1.0.0"
   }
 }

--- a/test.js
+++ b/test.js
@@ -19,3 +19,25 @@ test('alternate languages', async () => {
   const result = await lookup('Muñeca', 'es')
   expect(result.extract).toMatch('Una muñeca es una figura')
 })
+
+
+// cli tests
+const { spawn } = require('child_process')
+
+test('cli', async (done) => {
+  const reverse = spawn('node', ['cli.js', 'pomology'])
+  const chunks = []
+
+  reverse.stdout.on('data', (chunk) => {
+    chunks.push(chunk)
+  });
+
+  reverse.stdout.on('end', () => {
+    const output = Buffer
+      .concat(chunks)
+      .toString()
+
+    expect(output).toContain('Pomology:')
+    done()
+  });
+});

